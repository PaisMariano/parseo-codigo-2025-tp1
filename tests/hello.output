TOKEN_CLASS -> class
TOKEN_CREATE -> create
TOKEN_FEATURE -> feature
TOKEN_DO -> do
TOKEN_LPAREN -> (
TOKEN_STRING -> "Hola, Eiffel!%N"
TOKEN_RPAREN -> )
TOKEN_END -> end
TOKEN_END -> end
