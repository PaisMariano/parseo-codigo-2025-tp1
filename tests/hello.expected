TOKEN_CLASS -> class
TOKEN_IDENTIFIER -> HELLO_WORLD
TOKEN_CREATE -> create
TOKEN_IDENTIFIER -> make
TOKEN_FEATURE -> feature
TOKEN_IDENTIFIER -> make
TOKEN_DO -> do
TOKEN_IDENTIFIER -> print
TOKEN_LPAREN -> (
TOKEN_STRING -> "Hola, Eiffel!%N"
TOKEN_RPAREN -> )
TOKEN_END -> end
TOKEN_END -> end
